### PR TITLE
:fix fix an issue where adding chains in ethers doesn't work

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Description

* Fix an issue where adding chains in ethers doesn't work
* Remove ignored packages from changesets

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [ ] My changes generate no new warnings
- [ ] I have reviewed my own code
- [ ] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: packages/appkit-utils/src/ethers/EthersHelpersUtil.ts [hunk 1]
```diff
@@ -41,10 +41,14 @@ export const EthersHelpersUtil = {
       params: [
         {
           chainId: EthersHelpersUtil.numberToHexString(caipNetwork.id),
-          rpcUrls: caipNetwork.rpcUrls.default.http,
+          rpcUrls: [caipNetwork.rpcUrls.default.http[0]],
           chainName: caipNetwork.name,
-          nativeCurrency: caipNetwork.nativeCurrency,
-          blockExplorerUrls: caipNetwork.blockExplorers,
+          nativeCurrency: {
```
Comment (enesozturk) [inline packages/appkit-utils/src/ethers/EthersHelpersUtil.ts:46]: Are these happy with the Viem's type definitions?

Comment (enesozturk) [reply packages/appkit-utils/src/ethers/EthersHelpersUtil.ts:46]: I didn't get why the previous version not working fine 🤔 
```js
rpcUrls: caipNetwork.rpcUrls.default.http
```

Comment (svenvoskamp) [reply packages/appkit-utils/src/ethers/EthersHelpersUtil.ts:46]: I got this error : https://stackoverflow.com/questions/42376464/uncaught-domexception-failed-to-execute-postmessage-on-window-an-object-co

**Review by enesozturk — APPROVED** [review]

**Comment by enesozturk** [comment]
> Approver of this PR confirms that the changes are tested on the preview link
Which extension this was not working? All of them or a specific one? 

Also could you add some description to the PR - why this was needed what was the problem with extensions, what you found out etc? @svenvoskamp 🙏🏽

**Pull request merged**